### PR TITLE
Feature 5 player and realplayer

### LINKED ISSUE #4: [Feature 5] Player Object and RealPlayer subclass
Abstract player class and RealPlayer subclass. Players have a name and a Card array. Methods include drawCard(), playCard(), and endTurn().

### PR DESCRIPTION
**Pull request by paultran2**
Player and RealPlayer class

### CODE REVIEW

**Review by NonLan — DISMISSED** [review]
Thanks for making the classes, looks good! I'll implement the move functionality :)

**Review by SarahX43 — DISMISSED** [review]

**Review by kovalsviat — DISMISSED** [review]

File: src/main/java/entities/Player.java [hunk 1]
```diff
@@ -24,7 +26,9 @@ public void drawCards(List<Card> cards) {
         hand.addAll(cards);
     }
 
-    public void playCard(int n) {
+    public Card playCard(int n) {
+        Card toPlay = hand.get(n);
         hand.remove(n);
```
Comment (LogicalLunatic) [inline src/main/java/entities/Player.java]: I believe hand.remove(n) removes _and_ returns the card at index n. You can write this in line 29 in place of hand.get(n)

**Review by LogicalLunatic — CHANGES_REQUESTED** [review]
I noted something at line 30 of this commit that could make your code more optimized.

File: src/main/java/entities/Player.java (hunk 1 above)
Comment (paultran2) [reply src/main/java/entities/Player.java]: I changed it, thanks.

**Review by LogicalLunatic — DISMISSED** [review]

**Review by tomwurdfz — APPROVED** [review]
I think it's great and ready to be added to the main.
Everything looks good and it followed the concepts we discussed in class.

**Pull request merged**